src/main/java/com/theelixrlabs/healthcare/controller/DoctorController.java

```diff
@@ -48,21 +43,27 @@ public ResponseEntity<SuccessResponse<DoctorDto>> createDoctor(@Valid @RequestBo
     /**
      * Retrieves a doctor by their UUID.
      *
-     * @param id The ID of the doctor as a String, which will be converted to a UUID.
+     * @param doctorId    The ID of the doctor as a String, which will be converted to a UUID.
      * @return ResponseEntity containing a success response with the retrieved DoctorDto.
      */
-    @GetMapping(DoctorConstants.GET_DOCTOR_BY_ID_ENDPOINT)
-    public ResponseEntity<?> getDoctorById(@PathVariable(DoctorConstants.PATH_VARIABLE_DOCTOR_ID) String id) {
-        UUID doctorId;
-        try {
-            doctorId = UUID.fromString(id);
-        } catch (IllegalArgumentException illegalArgumentException) {
-            throw new CustomException(messageUtil.getMessage(MessageConstants.INVALID_UUID));
-        }
+    @GetMapping(DoctorConstants.DOCTOR_BY_ID_ENDPOINT)
+    public ResponseEntity<?> getDoctorById(@PathVariable(DoctorConstants.PATH_VARIABLE_DOCTOR_ID) String doctorId) throws CustomException {
         DoctorDto doctorDto = doctorService.getDoctorById(doctorId);
```

Comment:
Why return type is having "?" ResponseEntity<?> ?
Pls pass the correct response

Reply:
Resolved.